app/assets/templates/orders/history_item.jst.jade

```diff
@@ -0,0 +1,42 @@
+if locals.ordered_by_current_user
```

Comment:
issue:
It would be more DRY and readable to include in condition only the things there are different ;)
I'm not  a jade ninja, but I think (http://jade-lang.com/reference/interpolation) you can do just
`.big-order-box-#{locals.ordered_by_current_user ? 'red' : 'blue'}.animate__fade-in`